policies/release-policy.md

```diff
@@ -0,0 +1,123 @@
+  resolution.
+
+Any modifications to the release backlog should be conducted with an intent to
+preserve the balance. Thus, whenever a task is added, an equivalent effort lower
+priority task(s) must be removed. Conversely, when a task is removed, another
+may be added to the release backlog provided that such a task fits within the
+available time and is not prioritised higher than any existing task within the
+release backlog.
+
+This phase concludes with an alpha release readiness check by the steering
+committee. Upon successful completion of this check, an alpha release is
+created, marking the beginning of the Alpha phase.
+
+### 4. Release
+The Release phase of OpenSSL library commences with the creation of an alpha
+release at the end of the Development phase and spans over a period of six
+weeks. In the Release phase, no new features can be added, nor can any code
+refactoring be planned to be undertaken. Effective communication at this stage
+is crucial to ensure the quality of the release and, consequently, the success.
+
+  * **Alpha**:
+  During this phase, feature development is complete and feedback from the
+  community is being actively addressed. Testing, bug fixing, and documentation
+  updates are ongoing. Any critical issues discovered may lead to the removal of
+  the offending code.
+  * **Beta**:
+  This phase signifies the library is nearing its final release. Testing remains
+  intensive, but the focus is on ensuring stability. While all identified
+  regressions and critical concerns have been addressed, only crucial,
+  last-minute fixes are expected.
+  * **Final**:
+  The Final phase is focused on release preparation. The tasks involved include
+  updating the list of known issues, tagging the code git tree, and issuing
+  digital signatures among others. The phase concludes with the new release
+  becoming publicly available and communicated.
+
+See [Release Requirements Policy] for details on each type of release.
+
+### 5. Support[^3]
+  * **Full Support**:
+  Upon the final release a one-year Full Support is initiated for regular
+  releases, and a four-year period for LTS releases. During this phase, bugs and
+  security issues are addressed and fixed according to the [Stable Release
+  Updates Policy].
+  * **Maintenance Support**:
+  Immediately after the Full Support phase ends, the Maintenance Support phase
+  begins, lasting for one year. During this phase, the primary focus is on
+  fixing security issues, although other bugs may be addressed at the discretion
+  of OpenSSL engineering.
+
+[Stable Release Updates Policy]: https://www.openssl.org/policies/technical/stable-release-updates.html
+[Release Requirements Policy]: https://www.openssl.org/policies/technical/release-requirements.html
+
+[^1]: Feature Acceptance Criteria - 🚧 The document is a work in progress.
+[^2]: Release Steering Committee - A group comprised of four individuals: one
```

Comment:
I know we're initially planning on having four, but I wonder if we need to specify that explicitly in the policy like this. I'm not entirely sure four is actually enough to get a good representation from sufficient different types of people. It's ok to start off with four to see how it works out - but it would be nice if the policy allowed us to have more if needed.

Reply:
I have thought about this, but I haven't come up with another place where this can be specified. I will remove this limit to allow more community reps.

Reply:
Actually I think having this specified here is the **right** place as we can always update the policy as we learn from the process. I expect many updates over time and this should reflect the decision we are making **now**. A smaller focused group to start with on a rotational basis is the underlying principle here and is separate from the broader community engagement context.

This group isn't about going out an seeking community engagement from the wider community - it is about dealing with the focus of a specific release and that is much narrower than the item you are commenting on.

Reply:
I've given it further thought and replayed the scenarios in my mind. What's important here is to keep this group **small**. This doesn't mean we won't receive sufficient community input. Community members can provide their input by raising attention to items they care about that haven't received the 'next' label.